test/validate_exp.sh

```diff
@@ -81,14 +101,23 @@ for ext in "${exts[@]}"; do
         if grep "exists only in" "$diff_out" > /dev/null ; then
           cat "$diff_out"
           info "If this was an expected difference, you need to run tools/scripts/update_compiler_exp.sh"
-          fatal "* $(basename "$exp")"
+          something_failed
```

Comment:
Might be nice to pass `"$exp"` into `something_failed` and inject it into the error/success message there.

Reply:
That seems totally reasonable 👍